reanalysis/hail_filter_and_label.py

```diff
@@ -576,16 +578,12 @@ def extract_annotations(matrix: hl.MatrixTable) -> hl.MatrixTable:
             exac_ac_het=hl.or_else(matrix.exac.AC_Het, MISSING_INT),
             exac_ac_hom=hl.or_else(matrix.exac.AC_Hom, MISSING_INT),
             exac_ac_hemi=hl.or_else(matrix.exac.AC_Hemi, MISSING_INT),
-            gnomad_ex_cov=hl.or_else(matrix.gnomad_exome_coverage, MISSING_FLOAT_LO),
```

Comment:
The summary says you are removing the exac checks (which makes sense), but this looks like you are leaving exac in but dropping gnomad_exome (which makes much less sense)?

Reply:
Big ol' brainfart. 

The original MAF check was checking against gnomad genomes and exac. Should really be swapping out exac for gnomad exomes instead of removing gnomad exomes

Reply:
updated to check check gnomad exomes and genomes, instead of gnomad genomes + exac

also updated to check hemi & homozygote ACs in both gnomad exomes & genomes